game.py

```diff
@@ -0,0 +1,90 @@
+from etc.utils import Color, read_words, get_random_word, is_valid_word
+from collections import defaultdict
+from etc.consts import MAX_ATTEMPTS, INDEX_NOT_FOUND
+
+
+class WordleGame:
+    words = read_words()
+
+    def __init__(self):
+        self.__goal_word = ""
+        self.attempts = 0
```

Comment:
Why initialize if you're going to override in `reset`?

Reply:
To make its type known, should i make it None?

Reply:
I think so.. 🤷‍♂️